docs/ref/models/database-functions.txt

```diff
@@ -532,8 +532,8 @@ the Melbourne timezone (UTC +10:00), which changes the day, weekday, and hour
 values that are returned::
 
     >>> import pytz
-    >>> tzinfo = pytz.timezone('Australia/Melbourne')  # UTC+10:00
-    >>> with timezone.override(tzinfo):
+    >>> melb = pytz.timezone('Australia/Melbourne')  # UTC+10:00
+    >>> with timezone.override(melb):
```

Comment:
This does not lead to any exceptions.  I renamed it, because all other examples in the same file user `melb` an not `tzinfo` as variable name.